ca_gazebo/launch/create_2.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="cmd_vel" default="/cmd_vel" />
```

Comment:
I'm not sure about the need for cmd_vel arg since the client can remap this in their own launch file if including this one. But I think we should expose some useful gazebo_ros args here: `use_sim_time` and `world`. The last one I think the most useful since it's convenient for starting simulations in different worlds. See [jackal_gazebo/launchjackal_world.launch](https://github.com/jackal/jackal_simulator/blob/indigo-devel/jackal_gazebo/launch/jackal_world.launch) for a nice example.